File: QB_Challenges/PrepInsta/CPP/05_GO_Two_Nums/GOOTN_InBuilt.cpp
#include <iostream>
using namespace std;

void GOOTN(){
    int num1, num2;
    cout <<"Enter the first number: ";
    cin >>num1;
    cout <<"Enter the second number: ";
    cin >>num2;

    if (num1 == num2)
        cout << "both are equal";
    else
        cout << max(num1,num2) << " is greater ";

}

int main(){
    GOOTN();
    return 0;
}

File: QB_Challenges/PrepInsta/01_Getting_Started/05_GO_Two_Nums/Java/GOOTN_InBuilt.java
import java.util.Scanner;

public class GOOTN_InBuilt {
    public static void GOOTN() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter The First Number: ");
        int num1 = sc.nextInt();
        System.out.print("Enter The Second Number: ");
        int num2 = sc.nextInt();
        sc.close();

        int greatestNum = Math.max(num1, num2);
        // System.out.println(greatestNum);

        if(greatestNum == num1){
            System.out.println(greatestNum +" is greater than "+ num2);
        }else{
            System.out.println(greatestNum +" is greater than "+ num1);
        }
    }

    public static void main(String[] args) {
        GOOTN();
    }
}
